### ISSUE TITLE
problems with image plugin

### ISSUE THREAD

**Issue by sakulstra** [ngs/draft-js-markdown-shortcuts-plugin#30]
Hello I tried using this together with imagePlugin/dnd and alignment. Apparently it doesn't work :/ Images created via ![]() can't be focused, drag and dropped or aligned. Am I doing sth wrong or is this kind of combination not yet supported?
Best Regards,
Lukas

**Comment by mxstbr** [comment]
Yeah I'm running into this, even with the `dndPlugin` I can't seem to drag and drop images. (focussing them with the keyboard works though!)
